MinecraftClient/Resources/lang/de.ini

```diff
@@ -254,7 +254,8 @@ cmd.health.response=Gesundheit: {0}, Sättigung: {1}, Level: {2}, TotalExperienc
 # Inventory
 cmd.inventory.desc=Inventar-Befehl
 cmd.inventory.creative_done={0} x{1} im Slot #{2} angefordert.
-cmd.inventory.creative_fail=Item-Vergabe durch den Creative Modus wurde abgelehnt.
+cmd.inventory.creative_delete=Anfrage zum Leeren des Slots #{0} gesendet
+cmd.inventory.creative_fail=Failed to request Creative action
```

Comment:
```suggestion
cmd.inventory.creative_fail=Anfrage für Kreativ-Vorgang fehlgeschlagen
```

Reply:
@Daenges What do you think about "Kreativ-Vorgang"? Any other idea?

Reply:
Change "Vorgang" to "Aktion".